hardware/esp8266com/esp8266/libraries/ESP8266HTTPUpdateServer/src/ESP8266HTTPUpdateServer.cpp

```diff
@@ -0,0 +1,88 @@
+#include <Arduino.h>
+#include <WifiClient.h>
+#include <WifiServer.h>
+#include <ESP8266WebServer.h>
+#include <WifiUDP.h>
+#include "ESP8266HTTPUpdateServer.h"
+
+
+const char* ESP8266HTTPUpdateServer::_serverIndex = "<html><body><form method='POST' action='/update' enctype='multipart/form-data'>"
```

Comment:
Hmm, sorry, I missed that comment. It seems that My C++ is outdated :) , sounds good to me, will do.